src/manage_config_discovery.c

```diff
@@ -600,8 +600,6 @@ make_config_discovery (char *const uuid, char *const selector_name)
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.17586", "Brute force attacks");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.17584", "Brute force attacks");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.17368", "Brute force attacks");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.902799", "Databases");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.13859", "Denial of Service");
```

Comment:
https://secinfo.greenbone.net/omp?cmd=get_info&info_type=nvt&info_id=1.3.6.1.4.1.25623.1.0.13859&token=guest

DoS Vulnerability-Test and reporting for a "osTicket" product -> Not relevant for a discovery.